apps/lottie-preview/src/configUtils.ts

```diff
@@ -4,6 +4,16 @@ const JsonFieldType = 'Object';
 const AppWidgetNamespace = 'app';
 const DefaultWidgetId = 'objectEditor';
 
+// Batch configuration for API requests
```

Comment:
some of this is just guessing/was testing with using the AI and it worked for my 300 content types with JSON fields. any input here is definitely welcome

Reply:
Looks like for CMA calls it's 10 per second so batches of 10 does seem good